pkg/services/accesscontrol/acimpl/service.go

```diff
@@ -31,15 +32,21 @@ const (
 	cacheTTL = 10 * time.Second
 )
 
-func ProvideService(cfg *setting.Cfg, store db.DB, routeRegister routing.RouteRegister, cache *localcache.CacheService,
+func ProvideService(cfg *setting.Cfg, db db.DB, routeRegister routing.RouteRegister, cache *localcache.CacheService,
 	accessControl accesscontrol.AccessControl, features *featuremgmt.FeatureManager) (*Service, error) {
-	service := ProvideOSSService(cfg, database.ProvideService(store), cache, features)
+	service := ProvideOSSService(cfg, database.ProvideService(db), cache, features)
 
 	api.NewAccessControlAPI(routeRegister, accessControl, service, features).RegisterAPIEndpoints()
 	if err := accesscontrol.DeclareFixedRoles(service, cfg); err != nil {
 		return nil, err
 	}
 
+	if cfg.IsFeatureToggleEnabled(featuremgmt.FlagSplitScopes) {
```

Comment:
I do actually have an [enterprise PR](https://github.com/grafana/grafana-enterprise/pull/5490) as well, it just forgot to link it and to update the branch name so that it matches the OSS. Thanks for the reminder :)